resource/js/form.js

```diff
@@ -1,24 +1,15 @@
-// compute duration
 function manage_duration($scope) {
     if ($scope.traffic != undefined) {
```

Comment:
Should you check that `$scope.traffic` is > 0? I guess it is checked before but if you write : 
```javascript
if ($scope.traffic) {
```
It would do a second check plus what you wanted to do before. What do you think?

Reply:
done in https://github.com/dataiku/dss-plugin-ab-testing/pull/6/commits/fe62737a557effd806f29227fc5e561f63bcb3be